pkg/kubelet/kubelet.go

```diff
@@ -1668,11 +1753,18 @@ func (kl *Kubelet) tryUpdateNodeStatus() error {
 	// cAdvisor locally, e.g. for test-cmd.sh, and in integration test.
 	info, err := kl.GetCachedMachineInfo()
 	if err != nil {
-		glog.Errorf("error getting machine info: %v", err)
+		// TODO(roberthbailey): This is required for test-cmd.sh to pass.
```

Comment:
@smarterclayton, that exists?

On Fri, May 15, 2015 at 9:51 AM, Clayton Coleman <notifications@github.com>
wrote:

> In pkg/kubelet/kubelet.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6949#discussion_r30425249>
> :
>
> > @@ -1668,11 +1753,18 @@ func (kl *Kubelet) tryUpdateNodeStatus() error {
> >  	// cAdvisor locally, e.g. for test-cmd.sh, and in integration test.
> >  	info, err := kl.GetCachedMachineInfo()
> >  	if err != nil {
> > -		glog.Errorf("error getting machine info: %v", err)
> > +		// TODO(roberthbailey): This is required for test-cmd.sh to pass.
>
> Does this remove the ability of the administrator to set arbitrary
> capacity to a node via the API (if the kubelet always writes 0)?
> … <#14d587d3cb08fee9_>
> ----- Original Message -----
>  > @@ -1668,11 +1753,18 @@ func (kl *Kubelet) tryUpdateNodeStatus() error
> { > // cAdvisor locally, e.g. for test-cmd.sh, and in integration test. >
> info, err := kl.GetCachedMachineInfo() > if err != nil { > -
> glog.Errorf("error getting machine info: %v", err) > + //
> TODO(roberthbailey): This is required for test-cmd.sh to pass. +1 to a zero
> capacity being a sane default. cAdvisor should work in test-cmd in Linux
> builds (unless we're injecting a fake). --- Reply to this email directly or
> view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/6949/files#r30424260
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6949/files#r30425249>
> .
>